packages/labs/ssr/src/lib/render-lit-html.ts

```diff
@@ -527,13 +527,6 @@ export type RenderInfo = {
   deferHydration: boolean;
 };
 
-const defaultRenderInfo = {
```

Comment:
Hmm, it still seems like there's a circular dependency between the files; if this change fixed the WebPack issue, it seems likely it's because you moved this module-level variable (whose initializer depended on `LitElementRenderer`) into `render` (rather than depending on `renderValue` vs. `render`?

Reply:
Ah, you're right. Just moving the `defaultRenderInfo` into `render` alone does alleviate the error that webpack was throwing.

If we truly want to remove the circular dependency between files, I believe we do need to splice out `render` into its own module so that `lit-element-renderer` depends on `renderValue`, and `render` depends on both `lit-element-renderer` and `renderValue`.

That would be a breaking change since existing code, including our own example, dig into `lib/render-lit-html.js` for the `render` function. If it's going to be breaking like that, perhaps we might consider calling it `renderToString` or something a bit more descriptive, and also add `"exports"` into package.json to specify what's meant to be pubilc? `render` (`renderToString`?), `renderModule`, and dom shim related things?

Reply:
Seems ok to make the breaking change and update 11ty plugin accordingly.

Reply:
Was discussing with Al offline about this. It's really just a file name at this point but `render.js` feels pretty generic. The breaking change feels unnecessary if we just keep `render` function within `render-lit-html.js` and move everything else out to a `render-value.js`. Seems simpler then with no need to update the 11ty plugin either.